src/main/scala/mesosphere/marathon/api/akkahttp/Directives.scala

```diff
@@ -43,6 +44,24 @@ object Directives extends AuthDirectives with LeaderDirectives with AkkaDirectiv
     }
   }
 
+  def accepts(mediaType: MediaType): Directive0 = {
```

Comment:
Could you be so nice and use `accept` here https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/api/akkahttp/SystemController.scala#L37 as well? Thanks a ton.

Reply:
yes, I was actually planning to do that but in a separate PR once this one is approved (if you liked this approach - as it looks now you do :))

Reply:
Well, in general we don't need to explicitly filter accept headers as long as we have a proper marshaller. Akka http can do it transparently

Reply:
I can try to rewrite ping using marshallers. I think there it should work...

Reply:
I'd stay explicit here. I know that Akka should be able to do so. However, our API unfortunately does not seem to follow those rules exactly.